[2025-04-15 10:25:09] [ERROR] スクリプト実行中にエラーが発生しました: C:\kitting\OneDriveforBusinessManagementSystem\Incident_Management\SyncErrorCheck.ps1
例外タイプ: System.Management.Automation.RuntimeException
例外メッセージ: You cannot call a method on a null-valued expression.
位置: At C:\kitting\OneDriveforBusinessManagementSystem\Incident_Management\SyncErrorCheck.ps1:1080 char:93
+ … ect class='column-filter' data-column='$($columns.IndexOf($col))'>`n"
+                                            ~~~~~~~~~~~~~~~~~~~~~~
スタックトレース:
at <ScriptBlock>, C:\kitting\OneDriveforBusinessManagementSystem\Incident_Management\SyncErrorCheck.ps1: line 1080
at Invoke-SelectedScript, C:\kitting\OneDriveforBusinessManagementSystem\Main.ps1: line 343
at <ScriptBlock>, C:\kitting\OneDriveforBusinessManagementSystem\Main.ps1: line 393

